lib/aac/stream-types.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+module.exports = {
```

Comment:
Not sure if we need this file here, as it should be the same as m2ts stream-types